src/Test/Monad/Morph.hs

```diff
@@ -10,12 +10,20 @@ import Test.QuickCheck.HigherOrder (Equation(..))
 -- | Natural transformation.
 type m ~> n = forall t. m t -> n t
 
+-- |
+-- @
+-- hom (m '>>=' k) = hom m '>>=' hom . k
```

Comment:
This needs a natural-language description. Something like 'Natural transformations distribute over `(>>=)`.', perhaps?

Reply:
Monad morphisms are what distribute over `(>>=)`.

Reply:
"Commutativity" and "distributivity" often occur interchangeably in this context. "Monad morphisms commute with `(>>=)` and `return`".